### PR TITLE
Update seal for Market

### PR DESCRIPTION
**Pull request by grassdog**
We've renamed the Slack channel that the seal publishes to so this PR fixes that.

I think this is a decent time to reflect on a few things:

- Is the seal working for us?
- If so, should we prune (or add to) the repos it reports on?
- If not, should we remove it?

As it stands it prints out a pretty large message each day and given the number of repos it references I'm not sure that will change. Do folks find that these messages are prompts for them to review PRs?

### CODE REVIEW

**Comment by viraptor** [comment]
I wonder if the recent PRs are that interesting - usually we either ping each other/channel about them, or they're fresh in the stream of work. I'm mostly interested in PRs which are left over for a long time, which are on this list.

**Comment by grassdog** [comment]
Updating the seal so it sends reminders to a new dedicated channel #market-pull-requests.

We may replace the seal in the future though.

From @owenbutler in Slack:

> Seal could be useful, but has gone past noise and people are now trained to ignore it.
>
> My plan is to install https://www.github.com/probot/stale when I get a moment.  With the right thresholds, this will cut down on the noise.

File: config/envato.yml [hunk 1]
```diff
@@ -17,7 +17,7 @@ customer-group:
     - wip
 
 discovery:
-  channel: '#conversion-help'
+  channel: '#market-pull-requests'
```
Comment (grassdog) [inline config/envato.yml]: @geoffevason @madlep what do you folks think about routing the seal alerts for discovery to this new reminder channel?

Comment (madlep) [reply config/envato.yml]: I have no strong opinions, but sure 👍

Comment (geoffevason) [reply config/envato.yml]: @grassdog go for it. It'll add to the noise, but at least it'll be in a spot dedicated for that noise

Question - if someone is posted a pull request asking for reviews, where should they post it?  #market-pull-requests or #market-engineering ?

Comment (jaymiejones86) [reply config/envato.yml]: I feel that PR posts should be in #market-pull-requests but maybe cross post by copying the message to #market-engineering if it requires more visibility?

Comment (orien) [reply config/envato.yml]: Could we set up Github web hooks to automatically post PR links in this Slack channel?

Comment (grassdog) [reply config/envato.yml]: My opening pitch was for folks to continue to advertise PRs in #market-engineering and have #market-pull-requests as a place that robots remind us of open PRs. I've added that to the channel descriptions in Slack and to our docs https://docs.envato.net/market/#our-slack-channels

Happy to adjust as necessary. Mostly, I'd like to build a little momentum in bringing our engineering discussions for Market into #market-engineering.

**Pull request merged**